extensions/amp-facebook-comments/validator-amp-facebook-comments.protoascii

```diff
@@ -33,6 +33,7 @@ tags: {  # <amp-facebook-comments>
     name: "data-href"
     mandatory: true
   }
+  attrs: { name: "data-locale" }
```

Comment:
no need, `data-*` attributes are always white-listed. We only list them in validation rules if they need to be required or have a regex check their value.

Reply:
ditto other components

Reply:
Done